aries-site/src/examples/templates/persistent-navigation/MinimalSidebarExample.js

```diff
@@ -223,7 +210,7 @@ const PageContent = ({ activeItem }) => {
 
   return (
     pages && (
-      <Box fill overflow="auto">
+      <Box overflow="auto" flex>
```

Comment:
Is flex for Safari? Would a comment be good here?

Reply:
All browsers.